### ISSUE TITLE
StringShouldBeRawString false positive with prependIndent

### ISSUE THREAD

**Issue by TWiStErRob** [detekt/detekt#6145]
Might not be a false positive, let's see what you think!

## Context
In some cases when interpolating raw strings, we need to use multiple levels of indentation, instead of hacking with `.trimMargin()` (and its drawbacks), we can simply do this:
```kotlin
val x = """
    ...
""".trimIndent()
val usage = """
    ...
${x.prependIndent("\t\t\t\t")}
"""
```

```kotlin
val x = """
    ...
""".replaceIndent("\t\t\t\t")
```

## Expected Behavior
Above cases are not flagged for StringShouldBeRawString. I would say all usages of prependIndent/replaceIndent should be exempt, because if we replace it with raw, we lose the readability. We know that prepend/replace will very likely contain whitespace.

## Observed Behavior
Above codes are flagged.

## Your Environment

* Version of detekt used: 1.23.0
* Version of Gradle used (if applicable): 8.1.1
* Gradle scan link (add `--scan` option when running the gradle task): N/A
* Operating System and version: all
* Link to your project (if it's a public repository): N/A

---

Note: could the problem be that tabs shouldn't be flagged?, because this is also flagged `"Summary\t(total: 1)\t   1\t          0"` (an assertion on an aligned table)

**Comment by TWiStErRob** [comment]
Similar one: `val triplet = "\"\"\""` or `val TQ = "\"\"\""`, do we really want to encourage converting these to `"""""""""` 🤨?

**Comment by cortinico** [comment]
> I would say all usages of prependIndent/replaceIndent should be exempt

+1 on this

**Comment by atulgpt** [comment]
Hi, @TWiStErRob there is a config option `ignoredCharacters` will that solve your first issue? You have ignore the `\t` charcter

**Comment by TWiStErRob** [comment]
But in case of `"\tat foo\n\tat bar\n\tat qux"` it makes sense and that should be detected. It is a good rule, but there are some usages where it shouldn't be enforced, one such group is the stdlib indent methods, there might be more.

**Comment by TWiStErRob** [comment]
For `"""""""""` I don't even know what to do, probably just add an direct hard coded exception?
